### PR TITLE
Bump SQLitePCLRaw to 2.0.5-pre20210521085756

### PR DESCRIPTION
**Pull request by bricelam**

### CODE REVIEW

File: NuGet.config [hunk 1]
```diff
@@ -9,6 +9,7 @@
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
     <!-- Used for the Rich Navigation indexing task -->
     <add key="richnav" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-buildservices/nuget/v3/index.json" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```
Comment (bricelam) [inline NuGet.config:12]: @dotnet/aspnet-build What's the process for mirroring new versions from NuGet.org?

Comment (sebastienros) [reply NuGet.config:12]: We ask @wtgodbe, and magic happens

Comment (wtgodbe) [reply NuGet.config:12]: Presto, it's there

Comment (wtgodbe) [reply NuGet.config:12]: Remove this line

File: benchmark/EFCore.Sqlite.Benchmarks/EFCore.Sqlite.Benchmarks.csproj [hunk 2]
```diff
@@ -16,7 +16,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="SQLitePCLRaw.bundle_e_sqlite3" Version="2.0.5-pre20210119130047" PrivateAssets="None" />
+    <PackageReference Include="SQLitePCLRaw.bundle_e_sqlite3" Version="2.0.5-pre20210521085756" PrivateAssets="None" />
```
Comment (wtgodbe) [inline benchmark/EFCore.Sqlite.Benchmarks/EFCore.Sqlite.Benchmarks.csproj:19]: Why not use a property for the package version?

**Review by wtgodbe — CHANGES_REQUESTED** [review]
Remove the nuget.org line in nuget.config

File: benchmark/EFCore.Sqlite.Benchmarks/EFCore.Sqlite.Benchmarks.csproj (hunk 2 above)
Comment (bricelam) [reply benchmark/EFCore.Sqlite.Benchmarks/EFCore.Sqlite.Benchmarks.csproj:19]: Dependabot had issues when I tried that a while ago.

**Comment by wtgodbe** [comment]
Lots of onion peeling here on transitive dependencies

**Review by wtgodbe — APPROVED** [review]

**Comment by bricelam** [comment]
Thanks, Will! I'll be sure to include the closure of packages next time to make it easier.

**Pull request merged**